controllers/storagecluster/storagecluster_controller.go

```
@@ -69,6 +70,9 @@ func (r *StorageClusterReconciler) initializeImageVars() error {
 		err := fmt.Errorf("KUBE_RBAC_PROXY_IMAGE environment variable not found")
 		r.Log.Error(err, "Missing KUBE_RBAC_PROXY_IMAGE environment variable for ocs initialization.")
 		return err
+	} else if r.images.BlackboxExporter == "" {
+		err := fmt.Errorf("BLACKBOX_EXPORTER_IMAGE environment variable not found")
+		r.Log.Error(err, "Missing BLACKBOX_EXPORTER_IMAGE environment variable for ocs initialization.")
```

we should also return the error here